ex03.exs

```diff
@@ -51,12 +51,43 @@ defmodule Ex03 do
   """
 
   def pmap(collection, process_count, function) do
-    # your code goes here
     # I'm hoping to see a simple pipeline as the body of this function...
+    generate(collection)
+    |> split(process_count)
+    |> map(function)
+    |>List.flatten
   end
 
-  # and here...
 
+  def generate(collection) do
+    Enum.map(collection, fn val -> val end)
+  end
+
+  #Split up connection based on thread count
+  def split(collection, process_count) do
+    num = div(Enum.count(collection) , process_count)
```

Comment:
`div` here returns the wrong value. For example div(10, 4) returns a chunk size of 2, but it should be 3.

# -2